### PR TITLE
changes: def yourimc and def whileErrorNumbers

### PR DESCRIPTION
**Pull request by juang961225**
changes: def yourImc and def whileErrorNumbers
the function yourImc was added and this was added to the function that contains the while

### CODE REVIEW

File: ejcIMC.py [hunk 1]
```diff
@@ -25,6 +25,25 @@ def imcCalculate(statureValue, weigthValue):
     IMC = (weigthValue/(statureValue**2))
     return IMC
 
+def yourImc(imc):
+    valueImc = imc
```
Comment (Sonny97) [inline ejcIMC.py]: why u declarate valueImc if u dont use in the function?

File: ejcIMC.py [hunk 2]
```diff
@@ -25,6 +25,25 @@ def imcCalculate(statureValue, weigthValue):
...
+    if imc <= 16:
+        print('Su indice de masa corporal es ',imc ,': infrapeso e indica que se encuentra en delgadez severa')
+    elif imc <= 16.99:
+        print('Su indice de masa corporal es ',imc ,': infrapeso e indica que se encuentra en delgadez moderada')
+    elif imc <= 18.49:
+        print('Su indice de masa corporal es ',imc ,': infrapeso e indica que se encuentra en delgadez aceptable')
+    elif imc <= 24.99:
+        print('Su indice de masa corporal es ',imc ,': pso normal e indica que se encuentra en con un peso adecuado')
```
Comment (Sonny97) [inline ejcIMC.py]: revise the print message, it say ' ; pso normal' correct the "pso"'s word

File: ejcIMC.py [hunk 3]
```diff
@@ -25,6 +25,25 @@ def imcCalculate(statureValue, weigthValue):
...
+    elif imc <= 29.99:
+        print('Su indice de masa corporal es ',imc ,': sobre peso e indica que se encuentra en en un peso un poco encima de lo normal')
+    elif imc <= 34.99:
+        print('Su indice de masa corporal es ',imc ,': obeso e indica que se encuentra en el primer grado de obesidad')
+    elif imc <= 40.00:
+        print('Su indice de masa corporal es ',imc ,': obeso e indica que se encuentra en el segundo gado de obesidad')
```
Comment (Sonny97) [inline ejcIMC.py]: revise the print message, it say 'en el segundo gado de...', correct the "gado"'s word

File: ejcIMC.py [hunk 4]
```diff
@@ -34,6 +53,7 @@ def whileErrorNumbers(validation):
         value2 = numbersValidation(weigth)
         IMC = imcCalculate(value, value2)
         print('señor/señora', name, ' su indice de masa muscular es de: ', IMC )
+        yourImc(IMC)
         if validation == value:
```
Comment (Sonny97) [inline ejcIMC.py:54]: please change the position of this conditional, also make it in a function cuz u need validate value and value2

File: ejcIMC.py (hunk 1 above)
Comment (juang961225) [reply ejcIMC.py]: ok, i will delete valueImc

File: ejcIMC.py (hunk 2 above)
Comment (juang961225) [reply ejcIMC.py]: ok, i did solve print message

File: ejcIMC.py (hunk 4 above)
Comment (juang961225) [reply ejcIMC.py:54]: ok i did solve

**Comment by Sonny97** [comment]
why close the PR?... U need the approve to the another partners

**Pull request merged**